### ISSUE TITLE
error: unexpected argument '--dir' found

### ISSUE THREAD

**Issue by burg1ar** [solidiquis/erdtree#88]
config file: `~/.erdtreerc`, as follow:

```rc
--level 1
--icons
--scale 3

-s size
--dirs-first
```

run: `et SomeDir`

error occured:
```output
error: unexpected argument '--dir' found

  note: argument '--dirs-first' exists
```

Once I delete the `.erdtreerc` file, and run the equivalent command: `et -l 1 --icons --scale 3 -s size --dirs-first SomeDir`, everything works fine.

**Comment by solidiquis** [comment]
Hey @burg1ar, this was resolved in https://github.com/solidiquis/erdtree/pull/78.

It will be included in tonight/tomorrow's v1.6.0 release :]

**Comment by burg1ar** [comment]
thanks!
